fragment_info.go

```diff
@@ -570,6 +570,23 @@ func (fI *FragmentInfo) DumpSTDOUT() error {
 	return nil
 }
 
+// String retrieves the string representation of the FragmentInfo
+func (fI *FragmentInfo) String() string {
+	var tdbString *C.tiledb_string_t
+
+	ret := C.tiledb_fragment_info_dump_str(fI.context.tiledbContext, fI.tiledbFragmentInfo, &tdbString)
+	if ret != C.TILEDB_OK {
+		return fmt.Sprintf("Error dumping fragment info to string: %s", fI.context.LastError())
```

Comment:
Shouldn't we return an error here instead? Doesn't seem right that we only return a string but the string can contain an error.

Reply:
Done, thanks for taking a look